runtime.go

```diff
@@ -573,12 +573,8 @@ func getWorkingDir(basedir string, codeuri string) (string, error) {
 
 	dir := filepath.Join(abs, codeuri)
 
-	// ...but only if it actually exists
```

Comment:
If using docker-volume-basedir, because basedir is the path of the remote system, you will not know if basedir exists using os.Stat on the machine running aws-sam-local.